front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -146,13 +146,13 @@ const IdeasWithFiltersSidebar = ({
   );
 
   const openFiltersModal = useCallback(() => {
+    trackEventByName(tracks.openFiltersModalMobile);
```

Comment:
This function is gone after I merged my PR, so I fixed the merge conflict to solve that. :)